File: pl/0104.pl
for (1..9)
{
    if ($_ % 3 == 0)
    {
        print "$_, ";
    }
}
print "\n";

for (1..9)
{
    print "$_, " if ($_ % 3 == 0)
}
print "\n";

for (1..9)
{
    print "$_, " unless ($_ % 3)
}
print "\n";

File: rb/0104.rb
(1..9).each do |i|
    if i % 3 == 0
        print "#{i}, "
    end
end
print "\n"

(1..9).each do |i|
    print "#{i}, " if i % 3 == 0
end
print "\n"

(1..9).each do |i|
    print "#{i}, " unless i % 3 != 0
end
print "\n"
